orchagent/pfcwdorch.cpp

```diff
@@ -356,6 +356,32 @@ void PfcWdOrch<DropHandler, ForwardHandler>::deleteEntry(const string& name)
     SWSS_LOG_NOTICE("Stopped PFC Watchdog on port %s", name.c_str());
 }
 
+template <typename DropHandler, typename ForwardHandler>
+void PfcWdSwOrch<DropHandler, ForwardHandler>::createEntry(const string& key,
+        const vector<FieldValueTuple>& data)
+{
+    if (key == GENERAL)
+    {
+        for (auto valuePair: data)
+        {
+            const auto &field = fvField(valuePair);
+            const auto &value = fvValue(valuePair);
+
+            if (field == POLL_INTERVAL_FIELD)
```

Comment:
Is there a benefit of making this field global and not per port? If yes, what will happen in case if I will enable WD on some port(s) without setting global polling interval?

Reply:
Polling interval is global by design. It will use the default polling interval if not set specifically.